source/StarField.cpp

```diff
@@ -127,13 +127,20 @@ void StarField::SetHaze(const Sprite *sprite, bool allowAnimation)
 void StarField::Draw(const Point &pos, const Point &vel, double zoom) const
 {
 	double baseZoom = zoom;
+	int j = 1;
+
+	if(Preferences::Has("Parallax background"))
+		j = 4;
```

Comment:
```suggestion
	int passes = Preferences::Has("Parallax background") ? 4 : 1;
```
an alternate way to do the same thing
plz name variables in an explicit manner too, i and j doesnt tell much